### PR TITLE
BC-3872 Save and submit in one step

### PR DESCRIPTION
**Pull request by bergatco**
# Description

## Links to Tickets or other pull requests

https://ticketsystem.dbildungscloud.de/browse/BC-3872

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by CeEv — APPROVED** [review]

File: locales/de.json [hunk 1]
```diff
@@ -2086,7 +2086,9 @@
 			"importing": "importiere",
 			"importSubmission": "Abgabe des Schülers/ der Schülerin importieren",
 			"onTheSubject": "Zum Thema",
+			"saveDraft": "Entwurf anlegen",
```
Comment (bischofmax) [inline locales/de.json:2089]: Why is it "create" in german and "save" in english and other languages?

(As a User "Speichern" would tell me more about what is happening than "Anlegen" which is kind of vague for me.)

Comment (CeEv) [reply locales/de.json:2089]: @bergatco guter Punkt habe ich überlesen.

Comment (dyedwiper) [reply locales/de.json:2089]: I agree, that this wording is misleading. You can click the button several times and it always updates the draft, right? But the wording sounds like you would always a create a new one.

Comment (bergatco) [reply locales/de.json]: Fixed by renaming "Entwurf anlegen" to "Entwurf speichern"!

**Review by bischofmax — APPROVED** [review]

**Pull request merged**